### ISSUE TITLE
Render bus stations at lower zoom levels.

### ISSUE THREAD

**Issue by kolgza** [gravitystorm/openstreetmap-carto#4390]
The tag `amenity=bus_station` is used in two primary ways:

1. for busway stations along a BRT corridors
2. for transportation hubs in which  multiple bus routes converge [(the original intended use of this tag)](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbus_station)

Both uses of this tag suggests that `amenity=bus_station` describes important pieces of transportation infrastructure wherever it is used.
### Actual behavior

`amenity=bus_station` is rendered at z=15, the same zoom level in which `highway=bus_stop is rendered`.

### Expected behavior

`amenity=bus_station` should be rendered identically to how `railway=halt` is rendered.

**Comment by matkoniecz** [comment]
> amenity=bus_station should be rendered identically to how railway=halt is rendered.

You mean zoom level, not icon - right?

**Comment by kolgza** [comment]
yes

**Comment by jeisenbe** [comment]
I agree that this could make sense. @kolgza have you tested this out? Were you considering a PR?

**Comment by kolgza** [comment]
I had submitted a PR for this (#4391), but I've stopped working on it (actually, I've stopped working on all PRs).

**Comment by kolgza** [comment]
I'd like to copy @imagico's comment here:
> I am fairly critical of this for a number of reasons:
> 
> * it explicitly seems to be designed to cater a non-consensus overloading of the tag amenity=bus_station to map bus stops considered 'high importance' (for some essentially subjective or local context dependent definition of importance).
> 
> * reusing the bus stop symbol for bus stations at z13-z15 further supports mappers in the wrong assumption that bus stations are just high importance bus stops (which is not how the tag is predominantly used).
> 
> * it moves down the starting zoom level from z16 to z13 - three levels, which is a massive change in map design without discussing the effects this has on the map in general.  The balance of what we show and what we don't show is fairly delicate especially at the mid zoom levels so just starting to add some POI type three zoom levels down is going to significantly mess with that balance.
> 
>  * we typically start rendering labels for POI symbols one or two zoom levels after we start rendering the symbol.  Your change would mean the symbol starts at z13 while the label starts at z17 which would make it highly unique in this style.
> 
> 
> What would go a long way towards convincing me of the merits of this change is if you can show me how for normal use of the tag (as documented on the wiki) this change would have a positive effect on the map - in particular at relatively low latitudes where the map is dense.

https://github.com/gravitystorm/openstreetmap-carto/pull/4391#issuecomment-922447307

I don't really have a response to it.

**Comment by jeisenbe** [comment]
Looking at this more closely, `amenity=bus_station` is more similar to a light rail station, subway station or tram stop than a railway station. We render `railway=station` at z12 since these can be found in rural areas or small towns where they are the most important general map feature for many kilometers, and similarly `railyway=halt` is mostly a rural feature and starts at z13. In contrast, `station=subway` and `tram=stop` are not rendered till z14.

I would suggest testing `amenity=bus_station` with the smaller icon at z13 or z14

We should probably also wait to render `station=light_rail` and `station=funicular` till z13 or z14
